docs/Compile.rst

```diff
@@ -361,9 +361,10 @@ To install Chocolatey and the required dependencies:
 
 * Paste in the Chocolatey install command and hit enter
 * Close this ``cmd.exe`` window and open another Admin ``cmd.exe`` in the same way
-* Run the following command::
+* Run the following commands::
 
-    choco install git cmake strawberryperl -y
+    choco install git cmake.portable strawberryperl -y
+    cpanm XML::LibXML
```

Comment:
OK I understand now re cmake.portable - Chocolatey doesn't properly install the PATH for CMake. I didn't notice on my original test system because I'd had an old version of CMake installed and it as still in path.

So yeah the line changing cmake to cmake.portable is good.

But the cpanm line is definitely not needed - the whole reason to install StrawberryPerl is that it provides LibXML already, and running cpanm will just update the package by a small amount; not necessary.

Could you update the PR to just have the cmake -> cmake.portable edit.